### ISSUE TITLE
Too many annoying log while running the zoo/examples/openvino/predict.py

### ISSUE THREAD

**Issue by zhichao-li** [intel-analytics/analytics-zoo#1671]
Ideally,  we should add some doc to explain the exact meaning of the final output properly. 
```
pipeline-configs/object_detection/ssd_mobilenet_v1_0.75_depth_300x300_coco14_sync.config
pipeline-configs/object_detection/faster_rcnn_resnet101_ava_v2.1.config
pipeline-configs/object_detection/rfcn_resnet101_pets.config
pipeline-configs/object_detection/ssd_mobilenet_v1_ppn_shared_box_predictor_300x300_coco14_sync.config
pipeline-configs/object_detection/ssdlite_mobilenet_v2_coco.config
pipeline-configs/object_detection/ssd_mobilenet_v2_quantized_300x300_coco.config
pipeline-configs/object_detection/mask_rcnn_inception_resnet_v2_atrous_coco.config
pipeline-configs/object_detection/ssd_mobilenet_v1_focal_loss_pets.config
pipeline-configs/object_detection/ssdlite_mobilenet_v1_coco.config
pipeline-configs/object_detection/mask_rcnn_resnet101_atrous_coco.config
pipeline-configs/object_detection/ssd_mobilenet_v1_coco.config
pipeline-configs/object_detection/faster_rcnn_resnet101_kitti.config
pipeline-configs/object_detection/faster_rcnn_resnet101_fgvc.config
-rw-r--r--   1 lizhichao lizhichao 1.6K 10月 11 14:03 zoo-optimize-model-tf-image-classification-savedmodel.sh                                                                                    [176/7696]
-rw-r--r--   1 lizhichao lizhichao 1.7K 10月 11 14:03 zoo-optimize-model-tf-image-classification.sh                                                                                                         
-rw-r--r--   1 lizhichao lizhichao 1.3K 10月 11 14:03 zoo-optimize-model-tf-object-detection.sh                                                                                                             
[ INFO ] InferenceEngine: 0x7f7a9791d030                                                                                                                                                                    
Loading plugin                                                                                                                                                                                              
[ INFO ] Loading network files:                                                                                                                                                                             
        /tmp/ZooVino1507414553753561674/frozen_inference_graph.xml                                                                                                                                          
        /tmp/ZooVino1507414553753561674/frozen_inference_graph.bin                                                                                                                                          
[ INFO ] Loading model to the plugin                                                                                                                                                                        
[ INFO ] Start inference (1 iterations)                                                                                                                                                                     
                                                                                                                                                                                                            
Total inference time: 2706.73 ms                                                                                                                                                                            
Average running time of one iteration: 2706.73 ms                                                                                                                                                           
Throughput: 0.369449 FPS                                                                                                                                                                                    
                                                                                                                                                                                                            
JNI Total predict time: 2710.18 ms                                                                                                                                                                          
JNI Throughput: 0.368979 FPS                                                                                                                                                                                
                                                                                                                                                                                                            
[ INFO ] Start inference (1 iterations)                                                                                                                                                                     
                                                                                                                                                                                                            
Total inference time: 2657.24 ms                                                                                                                                                                            
Average running time of one iteration: 2657.24 ms                                                                                                                                                           
Throughput: 0.376331 FPS                                                                                                                                                                                    
                                                                                                                                                                                                            
JNI Total predict time: 2659.93 ms                                                                                                                                                                          
JNI Throughput: 0.375949 FPS                                                                                                                                                                                
                                                                                                                                                                                                            
[[[[ 0.         18.          0.99978775  0.1836277   0.16493058                                                                                                                                             
     0.6126913   0.88601446]                                                                                                                                                                                
   [-1.          0.          0.          0.          0.                                                                                                                                                     
     0.          0.        ]                                                                                                                                                                                
   [ 0.          0.          0.          0.          0.                                                                                                                                                     
     0.          0.        ]                                                                                                                                                                                
   [ 0.          0.          0.          0.          0.                                                                                                                                                     
     0.          0.        ]                                                                                                                                                                                
   [ 0.          0.          0.          0.          0.                                                                                                                                                     
     0.          0.        ]                                                                                                                                                                                
   [ 0.          0.          0.          0.          0.                                                                                                                                                     
     0.          0.        ]                                                                                                                                                                                
   [ 0.          0.          0.          0.          0.                                                                                                                                                     
     0.          0.        ]                                                                                                                                                                                
   [ 0.          0.          0.          0.          0.                                                                                                                                                     
     0.          0.        ]                                                                                                                                                                                
   [ 0.          0.          0.          0.          0.                                                                                                                                                     
     0.          0.        ]                                                                                                                                                                                
   [ 0.          0.          0.          0.          0.                                                                                                                                                     
     0.          0.        ]                                 
```

**Comment by qiyuangong** [comment]
Well, these logs are used for debugging. Will be removed when OpenVINO is stable enough. :)

**Comment by qiyuangong** [comment]
#1713

**Comment by qiyuangong** [comment]
https://github.com/intel-analytics/analytics-zoo/pull/1713 has been merged to master, and will backport to branch-0.6. Issue closed
